### PR TITLE
Update REAME.md for V 5.0.0

### PR DESCRIPTION
**Pull request by ogaca-dd**
Update README.md for next release 5.0.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,77 +99,9 @@ After the client is created, you can start sending events to your Datadog Event
...
-using (udP)
-{
-  // ...
-}
-// To disable the splitting of UDP messages, set this limit to 0
-```
+## Usage via the Statsd class
 
-A note about timing: Statsd will not attempt to handle any exceptions that occur in a timed method. If an unhandled exception is thrown while timing, a timer metric containing the time elapsed before the exception occurred will be sent or added to the send queue (depending on whether Send or Add is being called).
+`Statsd` is marked as obsolete as both `DogStatsdService` and `DogStatsd` methods do not block anymore and batch several metrics automatically in one UDP or UDS message.
```
Comment (truthbk) [inline README.md:104]: I am wondering if we should leave the snippet (or a simplified version of it) but say that it is deprecated for the reasons named above. I'm not sure, what do you think? 🤔

File: README.md [hunk 2]
```diff
@@ -45,10 +48,36 @@ Supported environment variables:
...
+};
+
+// Using DogStatsdService instance
+using (var service = new DogStatsdService())
+{
+    service.Configure(dogstatsdConfig);
+    service.Increment("example_metric.increment", tags: new[]{"environment:dev"});
+}  // Flush all metrics not yet sent
+
+// Or using DogStatsd static class
```
Comment (truthbk) [inline README.md:75]: Maybe two separate snippets (one for the service instance, one for the static class), so that there's less room for confusion?

Comment (truthbk) [reply README.md:75]: Also, should we include some examples for one or two additional metric types? gauges, histograms and distributions, for example? As we as a comment stating there are many other data types.

File: README.md (hunk 1 above)
Comment (ogaca-dd) [reply README.md:141]: I do not have a strong opinion on this topic but ideally we would like making `Statsd` private as soon as possible (to make changes to this class without doing breaking changes). Removing this section from README.md should reduce its usage and accelerate the migration.

**Pull request merged**